src/services/BookingService.ts

```diff
@@ -140,6 +142,17 @@ class BookingService {
       ));
     }
   }
+
+  public async deleteBookingsBySeveralSlotIds(slotIds: number[]): Promise<void> {
+    const repo = this.getRepository();
+    const affectedData = await repo.delete({ slot: { id: In(slotIds) } } );
+
+    if (affectedData.affected === 0) {
```

Comment:
Should we not be able to disable a room if there's no existing booking for any of its slot?